src/Directory.Build.props

```diff
@@ -1,8 +1,24 @@
 <Project>
     <PropertyGroup>
+        <LangVersion>8.0</LangVersion>
+        <TargetFrameworks>net6.0;net7.0</TargetFrameworks>
+
+        <Title>GitReleaseManager Tool</Title>
+        <Authors>GitTools Contributors</Authors>
+        <PackageProjectUrl>https://github.com/GitTools/GitReleaseManager</PackageProjectUrl>
+        <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
+        <PackageTags>github release notes create export</PackageTags>
+        <Description>Tool for creating and exporting releases for software applications from online Version Control Systems</Description>
```

Comment:
For the same reason as above, I am not sure that this one should be a common property.